src/components/utils/test/lock_posix_test.cc

```diff
@@ -60,48 +64,49 @@ TEST(LockPosixTest, CtorTestWithFalseArgument_ExpectNonRecursiveMutexCreated) {
   // Release mutex before destroy
   test_mutex.Release();
 }
-
-TEST(LockPosixTest, CtorTestWithTrueArgument_ExpectRecursiveMutexCreated) {
-  // Create Lock object
-  Lock test_mutex(true);
+TEST(LockPosixTest, AcquireMutex_ExpectMutexLocked) {
+  // Create Lock object (non-recursive mutex)
+  Lock test_mutex;
   // Lock mutex
   test_mutex.Acquire();
-  // Check if created mutex is recursive
-  EXPECT_TRUE(test_mutex.Try());
-  // Release mutex before destroy
-  test_mutex.Release();
+  // Try to lock it again. If locked expect false
+  EXPECT_FALSE(test_mutex.Try());
   test_mutex.Release();
 }
 
-TEST(LockPosixTest, AcquireMutex_ExpectMutexLocked) {
+TEST(LockPosixTest, TryLockNonRecursiveMutex_ExpectMutexNotLockedTwice) {
   // Create Lock object (non-recursive mutex)
   Lock test_mutex;
   // Lock mutex
-  test_mutex.Acquire();
+  test_mutex.Try();
```

Comment:
@AByzhynar , this logic not changed. just tests swapped places, and git dont know it.